changelogs/current.yaml

```diff
@@ -235,6 +235,11 @@ new_features:
   change: |
     added new field ``filter_metadata <envoy_v3_api_field_extensions.filters.http.ext_proc.v3.ExtProc.filter_metadata`` to aid in logging.
     Metadata will be stored in StreamInfo filter state under a namespace corresponding to the name of the ext proc filter.
+- area: access_log
```

Comment:
Didn't see where this is used - I assume you'll add more substitution formatters for these deltas?

Reply:
Yes, aiming that for a future PR, once this is approved and merged.